Makefile.am

```diff
@@ -28,6 +28,7 @@ EXTRA_DIST = \
 	package.json \
 	package-lock.json \
 	README.md \
+	pkg/lib/git-utils.sh \
```

Comment:
Is it actually important to ship this now, when we haven't so far? (FTR, I'm fine with it, just wondering if that is coincidental, or deliberate)